resources/lib/F1TVParser/AccountManager.py

```diff
@@ -117,29 +97,33 @@ def __createAuthorization__(self):
         if self.session_token is not None:
             # Try to load a cached social token
             try:
-                social_token_store = Store("app://tokens/social/{username}".format(username=self.username))
-                social_tokens = social_token_store.retrieve()
-                if len(social_tokens):
-                    for cached_token in social_tokens:
-                        cached_token_expiration_time = datetime.fromtimestamp(jwt.decode(cached_token, verify=False)['exp'])
-
-                        token_validity_time_remaining = cached_token_expiration_time - datetime.now()
-
-                        if token_validity_time_remaining.total_seconds() <= 60 * 60 * 24:
-                            self.__requestSocialToken()
-                        else:
-                            self.session.headers["Authorization"] = "JWT " + cached_token
+                cached_token = self.cache.get(f"app://tokens/social/{self.username}")
+                if cached_token is not None:
+                    cached_token_expiration_time = datetime.fromtimestamp(jwt.decode(cached_token, verify=False)['exp'])
+
+                    token_validity_time_remaining = cached_token_expiration_time - datetime.now()
+
+                    if token_validity_time_remaining.total_seconds() <= 60 * 60 * 24:
+                        self.__requestSocialToken()
+                    else:
+                        self.session.headers["Authorization"] = "JWT " + cached_token
                 else:
                     self.__requestSocialToken()
 
             except:
                 self.__requestSocialToken()
 
     def __init__(self):
-
-        self.auth_headers = {"CD-Language": "de-DE",
-                             "Content-Type": "application/json"}
+        self.cache = StorageServer.StorageServer("plugin.video.f1tv", 175316)
+        self.auth_data = {}
```

Comment:
On fresh installs of Kodi & the addon this causes an issue with loading because `exteractSessionData` tries to get the `cd-systemid` from in here.

Changing this line to `self.auth_data = {"cd-systemid": "0"}` fixed this.

Reply:
Sounds good - didn't test with a fresh install and I don't even think the system ID is used anywhere, just was present in the past so I left it in case it's needed